product_quantity_update_force_inventory/models/product.py

```diff
@@ -10,7 +10,7 @@ def action_update_quantity_on_inventory_adjustment(self):
         """Create an Inventory Adjustment instead of edit directly on quants"""
         self.ensure_one()
         view_form_id = self.env.ref("stock.view_inventory_form").id
-        action = self.env.ref("stock.action_inventory_form").read()[0]
+        action = self.env.ref("stock.action_inventory_form").sudo().read()[0]
```

Comment:
Why sudo() has been added? Just curious :eyes:

Reply:
I got an Action Window permission error and I found this was a way to fix it. I'm not sure if it's the best way, suggestions are welcomed:)